src/axelar/utils/Managed.sol

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import { IManaged } from '../interfaces/IManaged.sol';
+
+/**
+ * @title Managed
+ * @notice A contract module which provides a basic access control mechanism, where
+ * there is an account (an manager) that can be granted exclusive access to
+ * specific functions.
+ *
+ * The manager account is set through managership transfer. This module makes
+ * it possible to transfer the managership of the contract to a new account in one
+ * step, as well as to an interim pending manager. In the second flow the managership does not
+ * change until the pending manager accepts the managership transfer.
+ */
+abstract contract Managed is IManaged {
```

Comment:
might be overkill to have a separate contract for this. i'd be fine to keep this in the transceiver unless transferring ability is needed explicitly (outside upgrades). `Managed` could also be a bit confusing compared to the owner.

Reply:
removed